### ISSUE TITLE
To uninstall nix you will need to do the following things:

### ISSUE THREAD

**Issue by Kaleb47** [NixOS/nix#5152]
I tried to follow the advice below and I received this...

```rm: /nix: Read-only file system```

To uninstall nix you will need to do the following things:
* remove the `. "$HOME/.nix-profile/etc/profile.d/nix.sh"` line in your `~/.profile` or `~/.bash_profile`
* `rm -rf $HOME/{.nix-channels,.nix-defexpr,.nix-profile,.config/nixpkgs}`
* `sudo rm -rf /nix`
On a multi-user install, also remove all the `nixbld[0-9]` users

_Originally posted by @zimbatm in https://github.com/NixOS/nix/issues/1402#issuecomment-312496360_

**Comment by benkeil** [comment]
I guess there is missing a lot. what about the created volumes, launch daemon, files in /etc, ...?

**Comment by sontek** [comment]
Yeah, I'd like to see a supported uninstaller.    I don't mind installing with a bash script if some removal steps are well defined.  The whole reason people choose to use package managers like `homebrew` for pulling in tooling is so they have the option to clean it out if they'd like.

Can you please provide an uninstall option similar to the installer?

**Comment by sontek** [comment]
Yeah, what would be amazing.  Especially for recommending it to friends and family.   Feels scary to "try" nix when you can't get rid of it

**Comment by annymosse** [comment]
Actually I'm planning to remove uninstall it only coz it installed "32" new users!!!, what the hell for all of these users need!!!???.

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
